rebar.config

```diff
@@ -1,6 +1,9 @@
-{erl_opts, [debug_info, {parse_transform, lager_transform}]}.
+{erl_opts, [debug_info, {parse_transform, lager_transform},{lager_extra_sinks, [mylog]}]}.
 {deps, [{riak_core, "3.0.9", {pkg, riak_core_ng}},
-        {lager, "3.5.1"}]}.
+	{recon, "2.3.4"},
```

Comment:
We prefer not to add extra dependencies (part of the problem with other tutorials is that they had many dependencies so there where lots of spots that failed when they became outdated).